src/runtimeMain/kotlin/com/epam/drill/test/agent/TestListener.kt

```diff
@@ -40,12 +33,13 @@ actual object TestListener {
                 TestInfo::name.name to test,
                 TestInfo::startedAt.name to System.currentTimeMillis()
             )
```

Comment:
logger.info { "Test: $it STARTED" }
why did you remove the debug log?

Reply:
we set debug only for that artifact to adopt the project to not to turn on a trace in the agent. I don't think "test start" is debugging info

Reply:
1) we have been decided that change to INFO => logger.info { "Test: $it STARTED" } 
2) also we had a debug log with thread id etc. Why did remove it? It will be helpful when we investigate a bug in next time

Reply:
1. ok 
2. We used that logging to find a case of our problem, when we fixed it, we don't need that log anymore, those logs were only on one specific case, ofc I can back them if necessary, but don't think they can help us